src/components/gameScreen/components/customButton.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import {StyleSheet, TouchableOpacity} from 'react-native';
 import Animated, {useAnimatedStyle, useSharedValue, withSpring, withTiming} from 'react-native-reanimated';
+import {observer} from 'mobx-react-lite';
 import tick from '../../../assets/img/icons/tick.png';
-export const CustomButton = ({
-  color,
+import {bookmarkStore} from '../../../store/bookmarkStore';
+import {store} from '../../../store/store';
+export const CustomButton = observer(({
```

Comment:
it is not readable for human eye to see code which is not separated with empty lines. Code should be grouped by different parameters and separated with blank line:
1. imports
2. consts
3. states
and etc.